tico/utils/padding.py

```diff
@@ -40,8 +40,8 @@ def is_same_padding(
     if isinstance(padding, list):
         assert len(padding) == 2, "Padding should be a list of length 2."
 
-        input_HW = input_shape[1:2]  # N H W C
-        output_HW = output_shape[1:2]  # N H W C
+        input_HW = tuple(input_shape[1:3])  # N H W C
+        output_HW = tuple(output_shape[1:3])  # N H W C
```

Comment:
Without this patch, `ConvWithSamePadding2` is exported like below because list == torch.Size comparison doesn't work well. They should be casted into same data types.

![image](https://github.com/user-attachments/assets/da5bfef3-64c6-4eb9-8d9c-b9848a9a08e2)